docker/docker-compose-observability.yml

```diff
@@ -0,0 +1,44 @@
+version: "3"
+
+volumes:
+  prometheus_data: {}
```

Comment:
Can't we use a volume with a path ? Like it's done for the external services. This way, we could actually know where our data is.

Reply:
`The Prometheus image uses a volume to store the actual metrics. For production deployments it is highly recommended to use a named volume to ease managing the data on Prometheus upgrades.` from https://prometheus.io/docs/prometheus/latest/installation/